source/github-widgets/notice-bar.tsx

```diff
@@ -21,8 +21,11 @@ export default function addNotice(
 ): void {
 	select('#start-of-content')!.after(
 		<div className={`flash flash-full flash-${type}`}>
-			<div className="container-lg px-3 d-flex flex-items-center flex-justify-between">
-				<div>{message}</div> {action}
+			<div className="px-2">
```

Comment:
Can this element be dropped? `px-2` can probably go on the parent

Reply:
I copied the DOM GitHub uses when marking a PR ready for review.

Can't test now.